### PR TITLE
feat(tool): script to hash

### PR DESCRIPTION
**Pull request by Daryl-L**

### CODE REVIEW

**Comment by Danie0918** [comment]
It doesn't seem to work here, I got an error page after typing Code Hash and Args.

File: src/locales/zh.json [hunk 1]
```diff
@@ -901,7 +901,7 @@
       "address_conversion_description": "CKB2021 地址格式已从 bech32 变为 bech32m，我们建议开发人员在 dApp 中使用新格式。此工具可帮助您确定地址是已废弃还是新地址。",
       "address_rfc": "CKB 地址格式 RFC",
       "address_to_script": "Address to Script",
-      "script_to_address": "Script to Address",
```
Comment (Keith-CY) [inline src/locales/zh.json:904]: `script_to_address` is used at https://github.com/Magickbase/ckb-explorer-frontend/pull/316/files#diff-79c7cf7369a99998e79acd2e13e64307c13c662e0a5553226c211195930c5f0dR33

File: src/pages/Tools/AddressConversion/ScriptToHash.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+export const ScriptToHash: React.FC = () => {
+  const [codeHash, setCodeHash] = useState('')
+  const [args, setArgs] = useState('')
+  const [hash, setHash] = useState<string | undefined>(undefined)
+  const [hashType, setHashType] = useState(HashType.TYPE)
+  const { t } = useTranslation()
+
+  useMemo(() => {
+    try {
```
Comment (Keith-CY) [inline src/pages/Tools/AddressConversion/ScriptToHash.tsx:17]: Use `debounce` so the calculation is only done when the input is finished

Ref:
- debounce: https://www.freecodecamp.org/news/javascript-debounce-example/

File: src/pages/Tools/AddressConversion/ScriptToHash.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+          className={styles.input}
+          placeholder={`${t('tools.please_enter')} Args`}
+          value={args}
+          onChange={e => setArgs(e.target.value)}
+        />
+      </div>
+
+      {args !== '' && codeHash !== '' && (
+        <div className={styles.console} style={{ marginBottom: 16 }}>
+          <div>
```
Comment (Keith-CY) [inline src/pages/Tools/AddressConversion/ScriptToHash.tsx:60]: Unnecessary `div`

**Comment by Keith-CY** [comment]
Any updates for the conversations above @Daryl-L

**Comment by Danie0918** [comment]
cc @Daryl-L

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**